src/MDOP.jl

```diff
@@ -0,0 +1,103 @@
+using Neighborhood
+using StatsBase
+using Distances
+
+export beta_statistic
+export MDOP
+
+
+"""
+    MDOP(....)
+method by [^Nichkawde2013]
+
+
+[^Nichkawde2013]: Nichkawde, Chetan (2013). [Optimal state-space reconstruction using derivatives on projected manifold. Physical Review E 87, 022905](https://doi.org/10.1103/PhysRevE.87.022905).
+"""
+function MDOP(Y...)
+    #tbd
+
+end
+
+
+"""
+    beta_statistic(Y::Dataset, s::Dataset(::Array); kwargs...) → β
+Compute the β-statistic `β` for input phase space trajectory `Y` (of type
+`Dataset`) and a univariate time series 's' (1-dim. `Array` or `Dataset`)
```

Comment:
Here some changes I did were reverted. E.g. `s::Vector`.

Reply:
Hmpf, sorry. I deleted the wrong function with your corrected docstring. Now it should be okay again.